### DISCUSSION TITLE
Sunbird Ed Roadmap - 5.1 and 5.2

### DISCUSSION THREAD

**Discussion by Prateek-slokam** [Sunbird-Ed/Community#140] [General Discussion]
As a part of the Sunbird Ed development and release, I am starting this thread towards the roadmap discussions. I would like to invite you to the call today for Shikshalokam's contribution roadmap towards the next 2 releases, 5.1 and 5.2.
You can refer to the following for the same
[Shikshalokam Sunbird ED roadmap - EOY plan](https://www.google.com/url?q=https://docs.google.com/presentation/d/1RnHhcpjGxFzrAxhOSybs3C8k6kOaeUbdYOcYXWn7EQ0/edit?usp%3Dsharing&sa=D&source=calendar&ust=1663046632348111&usg=AOvVaw3crcCVHqycU4uF-9TXyb1y)
──────────

Date - 8th Sep, 3 pm - 4 pm
Join Zoom Meeting
[https://us06web.zoom.us/j/85639566091?pwd=QkJ0d3ZLNFVvY0paZWloeXMwMlY1Zz09](https://www.google.com/url?q=https://us06web.zoom.us/j/85639566091?pwd%3DQkJ0d3ZLNFVvY0paZWloeXMwMlY1Zz09&sa=D&source=calendar&ust=1663046632348111&usg=AOvVaw3qB2ZUFxpdJNViqK99Jxx_)

@sharathkashyap @swayangjit @rajeevsathish @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL @Prerna789 @aayushkedia95 @pkranga @pritamps @karn-nahata @madhucr @alok-os  @vijiurs @kiranharidas187 @Shakthieshwari @Preethi-Sl @coolbung @parthlawate @baljeetsingh @pramodkvarma @jagadishbabu

**Comment by luckynvdu** [comment]
Hello Everybody, please find the recording of the Sunbird ED roadmap (Shikshalokam) discussion below -

https://drive.google.com/file/d/15zX3eCA_fof5IID5BER28uks3Vmlbqhh/view?usp=sharing
